controllers/netbot.js

```diff
@@ -1222,6 +1223,21 @@ class netBot extends ControllerBot {
         });
         break;
       }
+      case "cpuProfile":{
+        (async () => {
+          const { applyProfileName, profiles} = value;
```

Comment:
We might need some comments here to describe input data structure for future reference